consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftRound.java

```diff
@@ -132,14 +139,19 @@ public void startRoundWith(
                     blockHeader, extraDataToPublish));
     final BlockHeader newHeader = headerBuilder.buildBlockHeader();
     final Block newBlock = new Block(newHeader, block.getBody());
+    LOG.debug(
+        "Created proposal from prepared certificate blockHeader={} extraData={}",
```

Comment:
appears to be 2 {}, but 3 params?

Reply:
nice catch. I was going to include the block number as well. don't think it's needed as this is included in the block header.
done.